src/executors/IifExecutor.js

```diff
@@ -4,6 +4,17 @@ import {getArgNames} from '../util/function';
 import {buildArgumentsFromRingaEvent} from '../util/executors';
 
 class IifExecutor extends ExecutorAbstract {
+  //-----------------------------------
+  // Constructor
+  //-----------------------------------
+  /**
+   * Constructor for a new IifExecutor.
+   *
+   * @param thread The parent thread that owns this command.
+   * @param condition The result of this function determines which executor to run
```

Comment:
Result is slightly ambiguous, let's say "A callback function whose return value can be true or false, determining whether to run the trueExecutor or the falseExecutor."